diffblue.yml

```diff
@@ -0,0 +1,3 @@
+buildCmd: mvn compile
```

Comment:
Please could you rebase on the latest master as this file should not have changed with this PR.